src/views/collections/art-of-accessibility/segments/hero.astro

```diff
@@ -0,0 +1,40 @@
+---
+import BookCover from "./book-cover.astro";
+import style from "./hero.module.scss";
+import BigFilledButton from "../../framework-field-guide/components/big-filled-button.astro";
+---
+
+<section aria-labelledby="book-title">
+	<div class={style.heroContainer}>
+		<div class={style.titleContainer}>
+			<h1 id="book-title" class={`text-style-headline-2 ${style.title}`}>
+				<span class={style.titleFirst}>The</span>
+				<span class={style.titleMiddle}>
+					<span class={style.titleMiddleHidden}>Accessibility</span>
```

Comment:
Todo: add aria-hidden to this

Reply:
This is required for the styling to work properly in many instances since `Art` and `Accessibility` are the wrong sizes